sdk/nginx/Azure.ResourceManager.Nginx/api/Azure.ResourceManager.Nginx.netstandard2.0.cs

```diff
@@ -196,6 +196,24 @@ protected MockableNginxSubscriptionResource() { }
 }
 namespace Azure.ResourceManager.Nginx.Models
 {
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ActivationState : System.IEquatable<Azure.ResourceManager.Nginx.Models.ActivationState>
```

Comment:
The name is too generic. Consider appending RP prefix or renaming it.
- use prepend-rp-prefix to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).

- use rename-mapping to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)